DockerHub/v2ray.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Get code
+CUR=${PWD}
+TAG=$(wget -qO- https://api.github.com/repos/v2fly/v2ray-core/tags | grep 'name' | cut -d\" -f4 | head -n1)
+git clone -b ${TAG} https://github.com/v2fly/v2ray-core.git && cd v2ray-core
+
+# Build release
+ARCHS=( 386 amd64 arm arm64 ppc64le s390x )
+ARMS=( 6 7 )
+for ARCH in ${ARCHS[@]}; do
+    if [ "${ARCH}" = "arm" ]; then
+        for ARM in ${ARMS[@]}; do
+            echo "Building v2ray-${ARCH}v${ARM}"
+            env CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GOARM=${ARM} go build -o v2ray -trimpath -ldflags "-s -w" ./main
+            env CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GOARM=${ARM} go build -o v2ctl -trimpath -ldflags "-s -w" -tags confonly ./infra/control/main
```

Comment:
Please follow the build command in here: https://www.v2fly.org/developer/intro/compile.html#%E5%A4%9A%E7%A7%8D%E6%9E%84%E5%BB%BA%E6%96%B9%E5%BC%8F